guides/micronaut-azure-http-functions/micronaut-azure-http-functions.adoc

```diff
@@ -84,28 +84,34 @@ You can invoke the GET route:
 [source,bash]
 ----
 curl -i localhost:7071/api/micronautguide
+----
+
+```
```

Comment:
I am not sure that this is proper asciidoc. I think they are Github fenced code blocks